api/functions/generate_enums.py

```diff
@@ -40,9 +41,54 @@ def create_enums(Table, enum_name, column):
 		enum_dict.update({select[1]: select[1]})
 
 	if not len(enum_dict):
-		return graphene.Enum(enum_name, 'EMPTY')
+		ret_enum = enum.Enum(enum_name, 'EMPTY')
 	else:
-		return graphene.Enum(enum_name, enum_dict)
+		ret_enum = enum.Enum(enum_name, enum_dict)
+
+	return graphene.Enum.from_enum(ret_enum)
+
+
+def create_enum_with_descriptions(Table, enum_name, enum_column, description_column):
```

Comment:
I've been adding the whole `:param:` and `:returns:` to the doc strings -- may be worth it to do the same!  :)